### ISSUE TITLE
Deleting a recorded session windows position on desktop

### ISSUE THREAD

**Issue by fa305757** [blindsidenetworks/moodle-mod_bigbluebuttonbn#208]
Using version 2.3.3 of the bigbluebutton moodle plugin when you need to delete a recorded session, the confirmation window is displayed at the bottom of the screen preventing the selection of yes or no options using the mouse.

**Comment by marcinmisiak** [comment]
Yes, there is bug. I can delete records of firefox 74 and chrome 80. Confirmation windows not appear.

**Comment by jfederico** [comment]
@fa305757 Do you still see this issue? Is this mobile or desktop?

While this should be addressed as a side effect once we replace YUI with AMD (work in progress), I haven;t been able to reproduce the issue on dektops.
